quickjs.c

```diff
@@ -29322,23 +29339,42 @@ static __exception int js_parse_export(JSParseState *s)
     switch(tok) {
     case '{':
         first_export = m->export_entries_count;
+        bool has_string_binding = false;
         while (s->token.val != '}') {
-            if (!token_is_ident(s->token.val)) {
-                js_parse_error(s, "identifier expected");
+            if (token_is_ident(s->token.val)) {
+                local_name = JS_DupAtom(ctx, s->token.u.ident.atom);
+            } else if (s->token.val == TOK_STRING) {
+                local_name = JS_ValueToAtom(ctx, s->token.u.str.str);
+                if (local_name == JS_ATOM_NULL) {
+                    return -1;
+                }
+                has_string_binding = true;
+            } else {
+                js_parse_error(s, "identifier or string expected");
                 return -1;
```

Comment:
Can be condensed to (here and elsewhere):
```suggestion
                return js_parse_error(s, "identifier or string expected");
```